modules/intensity_transform/misc/python/test/test_intensity_transform_bimef.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+# Python 2/3 compatibility
+from __future__ import print_function
+
+import os
+import cv2 as cv
+import numpy as np
+
+from tests_common import NewOpenCVTests, unittest
+
+class intensity_transform_test(NewOpenCVTests):
+    @unittest.skipIf('OPENCV_TEST_DATA_PATH' not in os.environ,
+                     "OPENCV_TEST_DATA_PATH is not defined")
+    def test_BIMEF(self):
+        filenames = ['P1000205_resize', 'P1010676_resize', 'P1010815_resize']
+
+        for f in filenames:
+            img = self.get_sample('cv/intensity_transform/BIMEF/{}.png'.format(f))
```

Comment:
Just an observation.
This is different from C++ `cvtest::findDataFile()` where `cv/`. is not needed.

Reply:
Right.
C++ test specifies prefix for its testdata globally [here](https://github.com/opencv/opencv_contrib/blob/4.3.0/modules/intensity_transform/test/test_main.cpp#L7).